### DISCUSSION TITLE
Minimal Platine

### DISCUSSION THREAD

**Discussion by beabel** [beabel/radiowecker#20] [Ideen / Ideas]
Ich glaube ich habe meinen Entwurf der Platine fertig.
Der Gedanke war in der Grundfläche gleich groß wie der TFT zu sein, 
jedoch sämtliche Leitungen bereits enthalten zu den beiden Max.
Alle Pins des ESP stehen nochmals jeweils 2 x zur Verfügung durch die äußeren Anschlüsse falls benötigt.
Entweder man lötet alles direkt auf (ESP,2 x Max, TFT) oder man nutzt diese Buchsen Leisten wodurch dann jedoch das ganze an Höhe gewinnt.

[EASYADA_Project](https://oshwlab.com/selfcommerce/mini_pcp)
[OPEN_PROJECT](https://easyeda.com/editor#project_id=62bb764e887142a29ddb728ef3af2fcd)

Falls mal jemand Zeit und Lust hat bitte mal drüber schauen, ob euch Fehler auffallen bevor ich die bestelle. :-)
![image](https://github.com/beabel/radiowecker/assets/12296286/47c9509b-70cd-43de-97e3-89a873bd25ea)
![image](https://github.com/beabel/radiowecker/assets/12296286/b142a9b4-e4c3-4c87-80b0-4c742deb551e)
![image](https://github.com/beabel/radiowecker/assets/12296286/4afba788-b0de-4d4f-91a7-753240afe970)

**Comment by brezel64** [comment]
Schöne Platine, meiner Meinung nach gehört vor den linken MAX98357A vor SD einen Widerstand von 150k zu Vin.
Habe das bei mir so drin und der Sound hört sich eher nach Stereo an.
Info dazu hier: https://www.az-delivery.de/blogs/azdelivery-blog-fur-arduino-und-raspberry-pi/max98357-was-ist-eigentlich-i2s

**Comment by beabel** [comment]
Falls bereits jemand diese Platine erstellen lassen hat, da war ein Fehler.
In der aktuellen Online Version bereits behoben.
Falls Ihr bereits fertige Platinen bestellt habt und nur ein Max funktioniert dann verbindet diese Pins: 
![image](https://github.com/beabel/radiowecker/assets/12296286/6e1aee95-9fb4-486c-b77e-e3b4b6bbaacd)

**Comment by beabel** [comment]
So, hier jetzt mal ein paar Bilder von der ganzen Geschichte:

Dies ist jetzt der Aufbau komplett zum stecken. man könnte auch alles direkt auf löten, damit würde es nochmals flacher werden der ganze Aufbau.
Und ich musste bei mir eben noch die oben erwähnte Brücke setzen. Im Online Project ist dies bereits behoben.
Aber ansonsten kein einziges Kabel zu löten, alles direkt integriert.
Größenvergleich:
![IMG20231205181134](https://github.com/beabel/radiowecker/assets/12296286/787d69e6-b7c3-4eab-9057-b7a37706d06a)
![IMG20231205181509](https://github.com/beabel/radiowecker/assets/12296286/480f120f-c765-4cfb-bb24-0993862e0539)
![IMG20231205181520](https://github.com/beabel/radiowecker/assets/12296286/8f691113-a0d5-49ce-a639-5926746f8069)

![005](https://github.com/beabel/radiowecker/assets/12296286/623567c0-79cc-484e-b013-fc6c914b5d07)
![004](https://github.com/beabel/radiowecker/assets/12296286/56665de3-3047-4e86-b94c-c1646a90f6ee)
![003](https://github.com/beabel/radiowecker/assets/12296286/0769f978-cd24-41ec-85e1-39d60ceedfda)
![002](https://github.com/beabel/radiowecker/assets/12296286/44adb204-8e41-4d7f-ad54-e796a0e52685)
![001](https://github.com/beabel/radiowecker/assets/12296286/d501ff8f-114b-44b5-933d-de3a94ce5431)

**Comment by beabel** [comment]
falls noch jemand die Platine verbessern möchte, man könnte noch einen Jumper einbauen um den Gain Wert je nach Anwendungsfall zu verändern. in der oben verlinkten Version ist Gain == 12db, dies könnte man flexibel gestalten.
Da es vielen ein wenig zu laut ist. Mir mittlerweile auch, aber den kurzen stört es nicht :-)
![image](https://github.com/beabel/radiowecker/assets/12296286/e8ce47c7-f689-4041-b0ed-28cae2c1656c)

**Comment by fraeggle** [comment]
@beabel Respect........ Super Arbeit.
Hab den "Bausatz" von AZ gestern bekommen und gleich zusammengebaut...

Fragen meinerseits über deine Platine......... Spannungsversorgung: Über den USB?
Wie befestigts du das in dem AZ Gehäuse?

Wo kann ich die Platine den käuflich erwerben? Über dich?

Das mit einstellbaren Gain find ich gar nicht schlecht. Wobei ich mit meinen Ohren auch schon nicht wirklich den Unterschied zw. 12db und 9db höre  ;-) Hab Gain aktuell offen  also 9db.


Gruß Peter

**Reply by beabel** [reply]
Du kannst direkt aus dem [Project](https://easyeda.com/editor#project_id=62bb764e887142a29ddb728ef3af2fcd) eine Bestellung ausführen.
Die BOM Datei enthält die Teile, welche dazu gehören, und die Gerberdatei ist die eigentliche Layout Datei.
steht dir dann frei, nur die Leiterplatte, oder direkt mit Bestückung.

Ich habe hier noch welche rumfliegen, jedoch noch die wo die Brücke gelötet werden muss aufgrund des kleinen Fehlers. (siehe [oben](https://easyeda.com/editor#project_id=62bb764e887142a29ddb728ef3af2fcd))
Kontakt: github@ne-xt.de

Und die Version mit Jumper habe ich nicht erstellt, sondern könnte jeder selbst in dem Project anpassen wenn benötigt, da ich aber im Moment keine weiteren benötige habe ich mich damit nicht beschäftigt.

in das original AZ Gehäuse passt es nicht. (vielleicht mit heißkleber ;-) dazu benötigst du ein anderes Gehäuse, jedoch habe ich weder einen Plan von Gehäuse erstellung, noch von 3D Druck, da hier noch nicht vorhanden.

Aber vielleicht findet sich ja jemand der eins erstellt.

Und ja Spannung nur über USB, habe bisher damit keine Probleme gehabt.

**Reply by AndreasCr** [reply]
Moin,
hat sich das mit dem 3D-Gehäuse geklärt ? 
Ansonsten würde ich mich da mal drangeben, wenn ich von Dir die nötige Platine bekommen kann ... könnten dann auch gemeinsam Idee in das Gehäuse einfließen lassen !
Gruß, vom Niederrhein, Andreas
